src/app/client/src/app/modules/groups/components/activity/activity-list/activity-list.component.ts

```diff
@@ -92,10 +103,17 @@ export class ActivityListComponent implements OnInit, OnDestroy {
     this.addTelemetry('confirm-remove-activity-button');
     const activityIds = [this.selectedActivity.identifier];
     this.showLoader = true;
+    this.showActivityList = false;
```

Comment:
Remove this showActivityList variable itself. This will be maintenance overhead. 
in HTML relay of this.activityList length to show/hide the div